openverse_catalog/dags/common/requester.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import time
+from collections.abc import Callable
```

Comment:
Ah, that should be [`typing.Callable`](https://docs.python.org/3/library/typing.html#typing.Callable)! (This change could) cause linting errors though, FWIW)
```suggestion
from typing import Callable
```

Reply:
Good catch! I originally went with your suggestion but the linter rewrote it to `collections.abc`. Should I make the change anyway?

Reply:
Woah 😮 which linter? That's definitely surprising...

Reply:
Okay I just tried this, for some reason `pyupgrade` is changing it?? That is so odd, let me look into it.

Reply:
Wow! That's what's [referenced in the documentation](https://docs.python.org/3/library/typing.html#callable)! Totally new to me, apologies! I'll go ahead and resolve this. ([Here's another issue for more context](https://github.com/PyCQA/isort/issues/1914))

Reply:
Ok cool! Yep, it was `pyupgrade`.